### ISSUE TITLE
Merge conflicts when sync

### ISSUE THREAD

**Issue by Pezhin** [PrairieLearn/PrairieLearn#1258]
```
Pull is not possible because you have unmerged files.
Please, fix them up in the work tree, and then use 'git add/rm <file>'
as appropriate to mark resolution, or use 'git commit -a'.
```
It happens when someone conducts a forced push and thus resulting in a merge conflicts.

Potential fix 
```
git fetch origin master
git reset --hard origin/master
```

**Comment by trombonekenny** [comment]
This has been fixed for the CS427 repo on PrairieLearn. We should investigate why PL couldn't pull in case we need to fix part of that process?

**Comment by Pezhin** [comment]
@trombonekenny : 
In my case, I was doing regular sync and realize there was a trivial problem (missing a 's') in my config file, so I did some edits and trying to use `git commit --amend` which was the start of the nightmare.  At this time, the repo on Github had a different hash with my "amended" commit and thus I conducted a forced push, which is the cause of merging conflict. 

To conclude, it was totally because of my habit of using `git commit amend`, and even it would result in blocking sync on the server side, it could be an issue that does not need a fix. (`wont fix` tags could be added). But it is something that probably requires to be documented or noted, as if this happens, course staff could not fix it independently.
